src/app/packages/core/services/generic-type-resolver.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injector, Injectable } from '@angular/core';
+import { GenericTypeResolver, InjectionResolverFilter } from './generic-type-resolver';
+
+@Injectable()
+export class GenericTypeResolverService {
+    constructor(private injector: Injector) { }
+
+    resolveMultiInjection(token: any, filterFunction: InjectionResolverFilter, isSingelton: boolean): any {
```

Comment:
I think that here you need default values also for filterFunction and  isSingelton